README.md

```diff
@@ -152,9 +152,11 @@ For Splunk Observability Cloud the following parameters are required:
 - `splunkObservability.accessToken`: Your Splunk Observability org access
   token.
 
-The following parameter is required for any of the destinations:
+The following parameter is required or optional depending on the Kubernetes service:
 
 - `clusterName`: arbitrary value that identifies your Kubernetes cluster. The value will be associated with every trace, metric and log as "k8s.cluster.name" attribute.
+  * Optional: EKS and GKE. If `clusterName` is specified it will overwrite detected value.
```

Comment:
```suggestion
  * Optional: If `distribution` is set to EKS, EKS/fargate, GKE, and GKE/autopilot. If `clusterName` is specified it will overwrite detected value.
```
Wasn't sure if EKS/fargate and GKE/autopilot were supported at a first glance.